### ISSUE TITLE
How to change the htaccess for Apache 2.4

### ISSUE THREAD

**Issue by trichers** [zikula/core#2938]
In Zikula are some htaccess files to protect the directory:

```
deny from all
<FilesMatch "\.(css|js|png|gif|jpg|bmp)$">
order allow,deny
allow from all
</filesmatch>
```

From Apache 2.4 the 

> Order deny,allow
> Deny from all

must be changed to

> Require all denied

How would be the ne notation for the code above?

**Comment by Kaik** [comment]
Apache 2.2
```
deny from all
<FilesMatch "\.(css|js|png|gif|jpg|bmp)$">
order allow,deny
allow from all
</filesmatch>
```
Apache 2.4
```
Require all denied
<FilesMatch "\.(css|js|png|gif|jpg|bmp)$">
Require all granted
</filesmatch>
```

**Comment by craigh** [comment]
Is this a core issue? is there something that needs to be changed in the code? Or is this simply a support request and now solved?

**Comment by Guite** [comment]
This is a core issue. We need to change all .htaccess files to make them work with both Apache 2.2 as well as Apache 2.4.

Example from https://wordpress.org/support/topic/htaccess-fix-for-apache-22-and-24

```
	# Apache 2.2
	<IfModule !mod_authz_core.c>
		Deny from all
	</IfModule>

	# Apache 2.4
	<IfModule mod_authz_core.c>
		Require all denied
	</IfModule>
```

**Comment by craigh** [comment]
from the topic you linked, it appears Wordpress implemented `<FilesMatch...` instead of the suggested `<Files...`. What do you recommend?

**Comment by Guite** [comment]
Fixed in #2982
